wilson.info.yml

```diff
@@ -36,6 +36,8 @@ install:
   - dynamic_page_cache
   - editor
   - editor_advanced_link
+  - environment_indicator
+  - environment_indicator_ui
```

Comment:
I think we should split the UI module out on local (we only need to set up the module on local, then export to config).

Reply:
Though Wilson includes the Config Split module, it doesn't currently contain any setup for the environments so I don't currently have a way to make this local only. I was thinking that by having the UI module on by default, when the site is installed you're in a better place to finish off the setup manually (i.e. putting in the remote environment URLs).